packages/desktop/components/popups/NodeInfoPopup.svelte

```diff
@@ -134,7 +138,13 @@
                     <Text type="p" fontSize="sm" fontWeight="font-600" secondary
                         >{localize(`${NODE_INFO_LOCALE_BASE_PATH}.${localeKey}`)}</Text
                     >
-                    {#if typeof nodeInfoValue === 'boolean'}
+                    {#if localeKey === 'metrics.referencedRate' || localeKey === 'metrics.referencedBlocksPerSecond' || localeKey === 'metrics.blocksPerSecond'}
+                        <Text type="p" fontSize="sm" secondary>
+                            {formatMetricWithOneDecimal(nodeInfoValue)}{localeKey === 'metrics.referencedRate'
+                                ? '%'
+                                : ''}
+                        </Text>
+                    {:else if typeof nodeInfoValue === 'boolean'}
```

Comment:
I wouldnt add the logic here, there is a section specifically to preprocess the data, in `processNodeInfoMapTab`

You could do something like 

```ts
function processNodeInfoMapTab(
    _nodeInfoTab: NodeInfoTab,
    key: string
): { localeKey: string; nodeInfoValue: unknown } {
    const nodeInfoTabObject = NODE_INFO_TAB_MAP[_nodeInfoTab]
    const localeKey = nodeInfoTabObject[key].localeKey
    let nodeInfoValue = ''
    if (key === 'url') {
        nodeInfoValue = node?.url
    } else {
        nodeInfoValue = resolveObjectPath(nodeInfo, nodeInfoTabObject[key]?.nodeInfoPath, null)
        if (key === 'referencedRate') {
            let referencedRate = Number(nodeInfoValue)
            if (referencedRate >= 0) {
                nodeInfoValue = `${formatNumber(Math.min(referencedRate, 100), 1, 1)}%`
            } else {
                nodeInfoValue = ''
            }
        }
    }

    return {
        localeKey,
        nodeInfoValue,
    }
}
```
With this you shouldnt need any HTML changes nor the extra custom function `formatMetricWithOneDecimal` which feels also a bit redundant